app/plugin.go

```diff
@@ -444,12 +448,14 @@ func (a *App) ServePluginRequest(w http.ResponseWriter, r *http.Request) {
 	r.Header.Del(model.HEADER_AUTH)
 	r.Header.Del("Referer")
 
+	params := mux.Vars(r)
+
 	newQuery := r.URL.Query()
 	newQuery.Del("access_token")
 	r.URL.RawQuery = newQuery.Encode()
+	r.URL.Path = strings.TrimPrefix(r.URL.Path, "/plugins/"+params["plugin_id"])
```

Comment:
It's a bit nicer for plugins to not worry about where their endpoint actually is and just be able to handle paths like "/webhook" instead of "/plugins/jira/webhook". They act a bit more like subrouters that way. I think this was my intention from the start and it's how I implemented the JIRA plugin without thinking, but I didn't really think about it when first implementing this. So this is why the JIRA plugin on pre-release is broken. 0.3/5 though.